pyapp/cookiecutter.json

```diff
@@ -12,9 +12,9 @@
     "dependabot_npm_interval": ["monthly", "weekly", "daily"],
     "dependabot_docker_interval": ["monthly", "weekly", "daily"],
     "devdata": ["no", "yes"],
-    "services": ["no", "yes"],
```

Comment:
I removed the `services` option. It doesn't make sense for a project to say "Yes, I want services." The project actually has to say _which_ services it wants, otherwise the cookiecutter doesn't know which services to put in `docker-compose.yml` and `ci.yml`.

So instead of the `services` option there are now `docker-compose/services.yml` and `.github/workflows/ci/services.yml` includes. Rather than a boolean option, the existence of these include files is what triggers the `docker-compose.yml` file and the `services` section in `ci.yml` to be generated.

Reply:
That's actually how it works currently. There's no need to ever retain an empty file. Take a look at https://github.com/hypothesis/cookiecutter-pypackage-test/pull/27/ for example: that PR just changes `"db"` to `"yes"` in `cookiecutter.json`, does _not_ add any include files, and re-runs `make template`, and all the services stuff (`docker-compose.yml` file etc) gets added to the project